pallets/avn-anchor/src/lib.rs

```diff
@@ -171,15 +179,24 @@ pub mod pallet {
         ChainId,
         Blake2_128Concat,
         CheckpointId,
-        H256,
-        ValueQuery,
+        CheckpointData,
+        OptionQuery,
     >;
 
+    #[pallet::storage]
+    pub type LatestCheckpoint<T: Config> =
```

Comment:
And this?

Reply:
Sorry not sure what this means. I can see it being inserted into the map but nothing else reads from it. We are effectively duplicating the data and I don't see a reason why.